### PR TITLE
Some small improvements to the comments

### PR DESCRIPTION
**Pull request by toncho11**
Helps with readability.

### CODE REVIEW

File: pyriemann_qiskit/classification.py [hunk 1]
```diff
@@ -428,16 +428,16 @@ class QuantumClassifierWithDefaultRiemannianPipeline(BaseEstimator,
                                                      ClassifierMixin,
                                                      TransformerMixin):
 
-    """Default pipeline wiht riemann geometry and quantum classifiers.
+    """Default pipeline wiht Riemann Geometry and a quantum classifier.
```
Comment (gcattan) [inline pyriemann_qiskit/classification.py:431]:
```suggestion
    """Default pipeline with Riemann Geometry and a quantum classifier.
```

File: pyriemann_qiskit/classification.py [hunk 2]
```diff
@@ -428,16 +428,16 @@ class QuantumClassifierWithDefaultRiemannianPipeline(BaseEstimator,
...
-    before applying quantum classification.
+    Projects the data into the tangent space of the Riemannian manifold
+    and applies quantum classification.
 
     The type of quantum classification (quantum SVM or VQC) depends on
     the value of the parameters.
 
     Data are entangled using a ZZFeatureMap. A SPSA optimizer and a two-local
-    circuirt are used in addition for VQC.
+    circuirt are used in addition when the VQC is selected.
```
Comment (gcattan) [inline pyriemann_qiskit/classification.py:440]:
```suggestion
    circuits are used in addition when the VQC is selected.
```

**Review by gcattan — APPROVED** [review]
Thanks @toncho11 !

**Review by qbarthelemy — APPROVED** [review]
Thx @toncho11

**Pull request merged**